test/E2ETests/Automation.CLI/CLITestBase.common.cs

```diff
@@ -17,8 +17,8 @@ public partial class CLITestBase
         private const string PackagesFolder = "packages";
 
         // This value is automatically updated by "build.ps1" script.
-        private const string TestPlatformCLIPackage = @"Microsoft.TestPlatform\17.2.0";
-        private const string VstestConsoleRelativePath = @"tools\net451\Common7\IDE\Extensions\TestPlatform\vstest.console.exe";
+        private const string TestPlatformCLIPackage = @"Microsoft.TestPlatform\17.4.0-preview-20220728-01";
```

Comment:
Not for this PR but could we read that version from the props file to avoid update of this source file everytime we update?

Reply:
Yep :)